backend/controllers/principle-controller.js

```diff
@@ -134,18 +206,20 @@ const requestResetPassword = async (req, res) => {
   try {
     const user = await principleService.findUser(email);
     if (!user) {
-      return res.status(404).json({ error: 'No user found with this email address' });
+      return res
+        .status(404)
+        .json({ error: 'No user found with this email address' });
     }
     // Generate a reset password token
     const passwordResetVerificationToken = await jwt.sign(
       { email },
       process.env.JWT_EMAIL_VERIFICATION_SECRET,
-      { expiresIn: '1h' },
+      { expiresIn: '1h' }
     );
```

Comment:
This value is used many times. It is possible to create a variable and reuse. That way, if this value needs to be changed, it will change in one place.